app/src/main/java/vunt/com/vn/moviedb_28/util/Constant.java

```diff
@@ -1,4 +1,8 @@
 package vunt.com.vn.moviedb_28.util;
 
 public class Constant {
+    public static final int NUMBER_1 = 1;
```

Comment:
em nên đặt là FIRST_PAGE
đặt theo cách sử dụng chứ không nên đặt theo giá trị
ví dụ 
```suggestion
    public static final int FIRST_PAGE = 1;
```